### PR TITLE
added avatar and header components

### PR DESCRIPTION
**Pull request by RohitDhatrak**

### CODE REVIEW

**Comment by Irfan-vit** [comment]
yep got it 👍

**Comment by kush-x7** [comment]
approved

**Comment by aman11s** [comment]
All good 🔥

**Comment by pavan6115** [comment]
looks good 👍

File: avatar.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* {
+}
```
Comment (RohitDhatrak) [inline avatar.css:2]: this is a comment

**Comment by pritam-kr** [comment]
👍

File: header.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* {
+}
```
Comment (RohitDhatrak) [inline header.css:2]: dsafdsfsd

**Review by Tarandeep-s1ngh — APPROVED** [review]
Approved

**Review by znjs — APPROVED** [review]

**Review by Meghana-sk — APPROVED** [review]

**Review by starver20 — APPROVED** [review]

File: avatar.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply avatar.css:2]: good

**Review by Ayushi20-19 — APPROVED** [review]

**Review by anandsr21harshit — APPROVED** [review]

**Comment by ragini07** [comment]
good

**Review by jayeshjain98 — APPROVED** [review]

**Review by aman11s — APPROVED** [review]

**Comment by manojasarsa** [comment]
Approved!!

**Pull request merged**